scootapi/setup/sched.go

```diff
@@ -0,0 +1,62 @@
+package setup
+
+import (
+	"log"
+)
+
+// SchedulerStrategy will startup a Scheduler (or setup a connection to one)
+type SchedulerStrategy interface {
+
+	// Startup starts up a Scheduler, returing the address of the server or an error
+	Startup() (string, error)
```

Comment:
Collapsing these makes it easier to do the wrong thing... Using one as the other will lead to some kind of trouble, so why make it easier?